jsonschema/tests/test_pre_commit_cli.py

```diff
@@ -0,0 +1,793 @@
+            stdout="===[SUCCESS]===(some_instance)===\n",
+            stderr="",
+        )
+
+    def test_successful_validation_of_just_the_schema(self):
+        self.assertOutputs(
+            files=dict(some_schema="{}", some_instance="{}"),
+            argv=["-s", "some_schema", "some_instance"],
+            stdout="",
+            stderr="",
+        )
+
+    def test_successful_validation_of_just_the_schema_pretty_output(self):
+        self.assertOutputs(
+            files=dict(some_schema="{}", some_instance="{}"),
+            argv=["--output", "pretty", "-s", "some_schema", "some_instance"],
+            stdout="===[SUCCESS]===(some_instance)===\n",
+            stderr="",
+        )
+
+    def test_successful_validation_via_explicit_base_uri(self):
+        ref_schema_file = tempfile.NamedTemporaryFile(delete=False)
+        self.addCleanup(os.remove, ref_schema_file.name)
+
+        ref_path = Path(ref_schema_file.name)
+        ref_path.write_text('{"definitions": {"num": {"type": "integer"}}}')
+
+        schema = f'{{"$ref": "{ref_path.name}#definitions/num"}}'
+
+        self.assertOutputs(
+            files=dict(some_schema=schema, some_instance="1"),
+            argv=[
+                "-s", "some_schema",
+                "--base-uri", ref_path.parent.as_uri() + "/",
+                "some_instance"
+            ],
+            stdout="",
+            stderr="",
+        )
+
+    def test_unsuccessful_validation_via_explicit_base_uri(self):
+        ref_schema_file = tempfile.NamedTemporaryFile(delete=False)
+        self.addCleanup(os.remove, ref_schema_file.name)
+
+        ref_path = Path(ref_schema_file.name)
+        ref_path.write_text('{"definitions": {"num": {"type": "integer"}}}')
+
+        schema = f'{{"$ref": "{ref_path.name}#definitions/num"}}'
+
+        self.assertOutputs(
+            files=dict(some_schema=schema, some_instance='"1"'),
+            argv=[
+                "-s", "some_schema",
+                "--base-uri", ref_path.parent.as_uri() + "/",
+                "some_instance"
+            ],
+            exit_code=1,
+            stdout="",
+            stderr="1: '1' is not of type 'integer'\n",
+        )
+
+    def test_nonexistent_file_with_explicit_base_uri(self):
+        schema = '{"$ref": "someNonexistentFile.json#definitions/num"}'
+        instance = "1"
+
+        with self.assertRaises(RefResolutionError) as e:
+            self.assertOutputs(
+                files=dict(
+                    some_schema=schema,
+                    some_instance=instance,
+                ),
+                argv=[
+                    "-s", "some_schema",
+                    "--base-uri", Path.cwd().as_uri(),
+                    "some_instance"
+                ],
+            )
+        error = str(e.exception)
+        self.assertIn(f"{os.sep}someNonexistentFile.json'", error)
+
+    def test_invalid_exlicit_base_uri(self):
```

Comment:
```suggestion
    def test_invalid_explicit_base_uri(self):
```

Reply:
This typo is also present in the test for official cli. You can submit a PR with a fix if you want to.